ai/github_issue_handling/usage.txt

```diff
@@ -0,0 +1,22 @@
+
+
+第一步：
```

Comment:
We can keep this Chinese document, but please add another English document. @Createsnow